config/routes.rb

```diff
@@ -147,6 +157,7 @@
   end
 
   get '/admin' => redirect('/admin/conferences')
-
+  conf = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc).first
+  get '/' => redirect("/conferences/#{conf.short_title}"), constraints: RootConstraint.new
```

Comment:
I tried it that way but it throws and error saying that a named route is being used for two different URLs. So I did it this way and it worked.